i18n/validate.py

```diff
@@ -222,6 +222,12 @@ def add_args(self):
             help="Includes empty translation strings in .prob files."
         )
 
+        self.parser.add_argument(
+            '-f', '--force',
+            action='store_true',
+            help="Check all files including merged files."
```

Comment:
Nit: I think the help text here could be a little clearer.  Maybe something like 'Validate all .po files, including those that are the product of a merge (see generate.py).' would be better?